catalogue/webapp/test/e2e/actions/common.ts

```diff
@@ -0,0 +1,41 @@
+export async function fillInputAction(
+  selector: string,
+  text: string
+): Promise<void> {
+  await page.fill(selector, text);
+}
+
+export async function pressEnterAction(selector: string): Promise<void> {
+  await page.press(selector, 'Enter');
+}
+
+export async function getInputValueAction(selector: string): Promise<string> {
+  const value = await page.$eval<string, HTMLInputElement>(
+    selector,
+    el => el.value
+  );
+  return value;
+}
+
+export function isMobile(): boolean {
+  return Boolean(deviceName);
```

Comment:
Where does `deviceName` come from?

Reply:
this is a global thing from jest-playwright . ive wrapped this round a function so we can change this later if api changes.